master-patient-index-client/src/test/java/gov/va/api/lighthouse/mpi/SoapMasterPatientIndexClientTest.java

```
@@ -171,13 +175,52 @@ void test1305Inspectors() {
         .getCorrespondingIdsResponseInspector(getCorrespondingIdsResponseInspector)
         .build()
         .request1305ByAttributes(attributes);
-
     verify(retrievePersonRequestInspector).accept(any());
     verify(retrievePersonResponseInspector).accept(any());
     verifyNoInteractions(getCorrespondingIdsRequestInspector);
     verifyNoInteractions(getCorrespondingIdsResponseInspector);
   }
 
+  @Test
+  @SneakyThrows
+  @EnabledIfSystemProperty(named = "interactive", matches = "true")
+  void test1309ForCerner() {
+    String MPI_URL = System.getProperty("mpi.url");
+    String MPI_KEYSTORE_PATH = System.getProperty("mpi.keystore-path");
+    String MPI_KEYSTORE_PASSWORD = System.getProperty("mpi.keystore-password");
+    String MPI_KEY_PASSWORD = System.getProperty("mpi.key-password");
+    String MPI_TRUSTSTORE_PATH = System.getProperty("mpi.truststore-path");
+    String MPI_TRUSTSTORE_PASSWORD = System.getProperty("mpi.truststore-password");
+    String MPI_USER_ID = System.getProperty("mpi.user-id");
+    String MPI_INTEGRATION_PROCESS_ID = System.getProperty("mpi.integration-process-id");
+    String MPI_AGENT_ID = System.getProperty("mpi.agent-id");
+    String CERNER_ID = System.getProperty("mpi.cerner-id");
```

Should these default to the mock-mpi versions of themselves?

---

Not sure the defaults would be helpful, but I could add them

---

That's fine. For an interactive test I'm not super worried about defaults